dashboard/app/models/school_district.rb

```diff
@@ -0,0 +1,48 @@
+# == Schema Information
+#
+# Table name: school_districts
+#
+#  id         :integer          not null, primary key
+#  name       :string(255)
+#  city       :string(255)
+#  state      :string(255)
+#  zip        :string(255)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class SchoolDistrict < ActiveRecord::Base
+  include Seeded
+
+  CSV_HEADERS =
+  {
+    :id => 'LEAID',
+    :name => 'NAME',
+    :city => 'LCITY',
+    :state => 'LSTATE',
+    :zip => 'LZIP'
+  }
```

Comment:
Where are the strings emanating from? A comment would be nice explaining the (my presumed) dependency on some school district listing?